libraries/plugins/delayed_node/delayed_node_plugin.cpp

```diff
@@ -105,6 +105,7 @@ void delayed_node_plugin::sync_with_trusted_node()
          fc::optional<graphene::chain::signed_block> block = my->database_api->get_block( db.head_block_num()+1 );
          FC_ASSERT(block, "Trusted node claims it has blocks it doesn't actually have.");
          ilog("Pushing block #${n}", ("n", block->block_num()));
+         db.precompute_parallel( *block, graphene::chain::database::skip_nothing ).wait();
```

Comment:
Can adopt same approach used in reindex here for even better performance, IMHO low priority though. Add a TODO?

Reply:
added